src/overlays/actors/ovl_En_Sth/z_en_sth.c

```diff
@@ -40,45 +55,357 @@ static ColliderCylinderInit D_80B0B404 = {
     },
     { 30, 40, 0, { 0, 0, 0 } },
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Sth/func_80B07540.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Sth/EnSth_Init.s")
+static s16 sObjectIds[6] = {
+    OBJECT_AHG, OBJECT_BOJ, OBJECT_BOJ, OBJECT_BOJ, OBJECT_BOJ, OBJECT_BOJ,
+};
+
+static FlexSkeletonHeader* sSkeletons[6] = {
+    0x060000F0, 0x060000F0, 0x060000F0, 0x060000F0, 0x060000F0, 0x060000F0,
+};
```

Comment:
lol, such a necessary array. Though shouldn't these be replaced with their proper symbols.

Reply:
these objects files for these arent "decompiled" yet, ill probably do them soon and theyll either make it into this PR before it gets merged, or if not they should make it into my "various objects" PR